docs/HyperIndex/Guides/event-handlers.mdx

```diff
@@ -123,21 +88,17 @@ Below is a list of raw event information that can be accessed:
 
 ### Context
 
-Handler functions can access the loaded entities that were loaded in the handler via
+In your handler functions you can access entities in the database via the asyncronous 'get' function as shown below:
 
 ```javascript
-context.<EntityName>.load(<id>)
+await context.<EntityName>.get(<id>)
 ```
 
-they can access these loaded entities via
-
-```javascript
-context.<EntityName>.get(<id>)
-```
+The context also contains some logging functions that include the context automatically. You can also use `console.log`.
```

Comment:
For me, it's not clear why I should use the logging functions on the context instead of `console.log.` And an example would be nice.

Reply:
```suggestion
The context also contains some logging functions that include the context to improve the debugging experience, such as the block and event that is currently being processed automatically. You can also use `console.log` if you don't benefit from that extra context.
```